RELEASE_NOTES.md

```diff
@@ -2,6 +2,7 @@
 
 ## 3.1.0 (planned)
 - NLRFIM-109: Logging for Java OIO SAML 3
+- NLRFIM-120: Support SOAP bindings in Java OIO SAML 3
```

Comment:
Technically this is planned for 3.2.0 - we should build a 3.1.0 release and publish it.